projects/demo-integrations/src/tests/component-testing/predicate/synchronous-test-input.component.ts

```diff
@@ -0,0 +1,54 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MaskitoDirective} from '@maskito/angular';
+import {MaskitoElementPredicate, MaskitoOptions} from '@maskito/core';
+
+@Component({
+    standalone: true,
+    selector: 'synchronous-test-input',
+    imports: [MaskitoDirective, FormsModule],
+    template: `
+        <div
+            [maskito]="card.matches(':focus') ? cardMask : nameMask"
+            [maskitoElement]="card.matches(':focus') ? cardPredicate : namePredicate"
+        >
+            <input
+                #card
+                [(ngModel)]="value.number"
+            />
+            <input [(ngModel)]="value.name" />
+        </div>
+    `,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SynchronousTestInputComponent {
```

Comment:
Why `Synchronous`?

May be something like `MultiTestInput`?

Reply:
Because the other test in asynchronous. I renamed it.